### ISSUE TITLE
[format.string.std] Should "UCS" be replaced with "Unicode"?

### ISSUE THREAD

**Issue by JohelEGP** [cplusplus/draft#6155]
#6144 adds a new use of "UCS scalar value". #6126 changed all occurrences of it to "Unicode scalar value". Should this added "UCS" also be changed to "Unicode"?

#6144 also adds some uses of "Unicode encoding". Although #6126 changed many of those, it left some (reproduced below), so perhaps the added ones can stay.

```terminal
$ git checkout motions-2023-02-cwg-8
$ git grep 'Unicode encoding' *.tex
future.tex:  arguments for these functions. How Unicode encoding conversions are performed is
lex.tex:for the respective Unicode encoding form.
lex.tex:No character lacks representation in any Unicode encoding form.
time.tex:If the string literal encoding is a Unicode encoding form and
```

**Comment by jensmaurer** [comment]
Yes to the first.

The second needs further inspection.

"No character lacks representation in any Unicode encoding form."  is a true statement for any Unicode encoding form (also those beyond UTF-8,16,32).

**Comment by JohelEGP** [comment]
Resolved by #6144 itself.
